src/DynamoPackages/PackageManagerSearchElement.cs

```diff
@@ -64,6 +64,11 @@ public string HostsString
                         hostsString += host + "  ";
                     }
                 }
+                //added to this dummy package for representation, to be removed and replace space at the end with newline above
+                if (Name == "dronovBIM_v2")
+                {
+                    hostsString += "CPython3"+ Environment.NewLine + "IronPython2"+ Environment.NewLine + "Revit";
```

Comment:
@QilongTang  Here `HostsString` is derived from `host_dependencies` that we are using  to display dependencies, including python deps.

Reply:
Avoid string literals wherever possible. Makes it easy to change it.